benchexec/container.py

```diff
@@ -115,6 +115,12 @@
 )
 """Whether we use generated native code for clone or an unsafe Python fallback"""
 
+_ERROR_MSG_USER_NS_RESTRICTION = (
+    "Ubuntu 24.04 restircts unprivileged user namespaces, please try "
+    "'echo 0 | sudo tee /proc/sys/kernel/apparmor_restrict_unprivileged_userns' "
```

Comment:
The standard way of changing sysctls is with `sysctl -w ...`.
This would be `sysctl -w kernel.apparmor_restrict_unprivileged_userns=0` here.

We could also use the same style of error message as we have for similar cases, for example we already have this:
> Unprivileged user namespaces forbidden on this system, please enable them with 'sysctl -w kernel.unprivileged_userns_clone=1' or disable container mode

Reply:
I have changed the error message to this:
```Python
_ERROR_MSG_USER_NS_RESTRICTION = (
    "Unprivileged user namespaces forbidden on this system, please "
    "enable them with 'sysctl -w kernel.apparmor_restrict_unprivileged_userns=0', "
    "Ubuntu disable it by default since 24.04, refer to "
    "https://ubuntu.com/blog/ubuntu-23-10-restricted-unprivileged-user-namespaces "
    "for more information."
)
```
 I noticed that `container.md` already has an entry for `Cannot execute ...: Creating namespace for container mode failed` and a link to `INSTALL.md#kernel-requirements` followed. We could just add the solution for Ubuntu 24.04 in `INSTALL.md#kernel-requirements`, and leave `container.md` unchanged. Users should be able to find the solution by following the error message.